src/main/java/org/prebid/server/proto/openrtb/ext/request/yearxero/ExtImpYearxero.java

```diff
@@ -0,0 +1,16 @@
+package org.prebid.server.proto.openrtb.ext.request.yearxero;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.util.Map;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class ExtImpYearxero {
```

Comment:
looks like this class isn't used anywhere, so it could be removed

Reply:
Yes, this class is not used directly anywhere. But I thought that we need to declare such classes for custom extensions in order to make using of our adapter more clear, am I right?

Reply:
No, it's not necessary at all while the adapter just passes through everything as-is. So in order to have the code clean, I suggest removing parts that are not used.

Reply:
Done, I removed the file.